d2bs/kolbot/libs/bots/MFHelper.js

```diff
@@ -155,9 +159,25 @@ MainLoop:
 
 			if (command !== oldCommand) {
 				oldCommand = command;
-
+				
 				if (command.indexOf("quit") > -1) {
 					break MainLoop;
+				} else if (command.indexOf("goto") > -1) {
+					print("ÿc4MFHelperÿc0: Goto");
+					split = command.split("goto A")[1];
```

Comment:
Since you just want the second part of the string and dont do anything with the value first part.
`split = command.substr(6)` might be better and quicker

Reply:
updated this one